docs/cmdlets/Get-OutputBinding.md

```diff
@@ -0,0 +1,88 @@
+---
+external help file: Microsoft.Azure.Functions.PowerShellWorker-help.xml
+Module Name: Microsoft.Azure.Functions.PowerShellWorker
+online version:
+schema: 2.0.0
+---
+
+# Get-OutputBinding
+
+## SYNOPSIS
+Gets the hashtable of the output bindings set so far.
+
+## SYNTAX
+
+```
+Get-OutputBinding [[-Name] <String[]>] [-Purge] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Gets the hashtable of the output bindings set so far.
+
+## EXAMPLES
+
+### EXAMPLE 1
+```
+Get-OutputBinding
+```
+
+Gets the hashtable of all the output bindings set so far.
+
+### EXAMPLE 2
+```
+Get-OutputBinding -Name res
+```
+
+Gets the hashtable of specific output binding.
+
+### EXAMPLE 3
+```
+Get-OutputBinding -Name r*
+```
+
+Gets the hashtable of output bindings that match the wildcard.
+
+## PARAMETERS
+
+### -Name
+The name of the output binding you want to get.
+Supports wildcards.
+
+```yaml
+Type: String[]
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: 1
+Default value: *
+Accept pipeline input: True (ByPropertyName, ByValue)
+Accept wildcard characters: False

```

Comment:
this is false but it _does_ support wildcard characters?

Reply:
A bug in PlatyPS? I don't think it's able to figure out if a parameter accepts wildcard.

Reply:
Cannot change it manually, unless we revert the doc check code in build.